acceptance-tests/test/end-to-end/helpers/misc.js

```diff
@@ -51,6 +51,7 @@ async function multipleSearch(CaseSearch, I, searchOption) {
   }
 
   I.wait(CCPBATConstants.fiveSecondWaitTime);
+  await bulkScanApiCalls.toggleOffCaseValidation();
```

Comment:
@VamshiRudrabhatlaHMCTS Can you please remove this step as it affects all tests? And we're removing this feature from the pay bubble as it's creating a lot of issues in case validation.